main.go

```diff
@@ -15,20 +15,27 @@ import (
 
 
 func main() {
-	db, err := sql.Open("postgres", "host=localhost port=5432 user=postgres password=Dm2016dM dbname=PadawanChat sslmode=disable")
+	dbConnection, err := sql.Open("postgres", "host=localhost port=5432 user=postgres password=Dm2016dM dbname=PadawanChat sslmode=disable")
 	if err != nil {
 		fmt.Printf("Error: %v\n", err)
 		os.Exit(1)
 	}
 
-	dbStorage := ports.NewDbStorage(db)
-	usecase := usecases.NewMessageUsecase(dbStorage)
-	handler := handlers.NewMessageHadler(usecase)
+	messageStorage := ports.NewMessageStorage(dbConnection)
+	authorStorage := ports.NewAuthorStorage(dbConnection)
+	messageUsecase := usecases.NewMessageUsecase(messageStorage, authorStorage)
+	authorUsecase := usecases.NewAuthorUsecase(authorStorage, messageStorage)
+	messageHandler := handlers.NewMessageHadler(messageUsecase)
+	authorHandler := handlers.NewAuthorHandler(authorUsecase)
 
 	mux := mux.NewRouter()
-	mux.HandleFunc("/message/create", handler.MessageCreate).Methods("POST")
-	mux.HandleFunc("/message/delete/{id}", handler.MessageDelete).Methods("DELETE")
-	mux.HandleFunc("/message/get/all", handler.MessageGetAll).Methods("GET")
+	mux.HandleFunc("/message/create", messageHandler.MessageCreate).Methods("POST")
+	mux.HandleFunc("/message/delete/{id}", messageHandler.MessageDelete).Methods("DELETE")
```

Comment:
```suggestion
	mux.HandleFunc("/message/{id}/delete", messageHandler.MessageDelete).Methods("DELETE")
```
Its common to use pattern entity/{entityID}/{some operation}